addons/explosives/CfgAmmo.hpp

```diff
@@ -118,6 +119,9 @@ class CfgAmmo {
         GVAR(magazine) = "IEDUrbanBig_Remote_Mag";
         GVAR(size) = 1;
         soundTrigger[] = {"A3\Sounds_F\weapons\mines\mech_trigger_1", 0.8, 1, 40};
+        GVAR(ringtones)[] = {{QPATHTOF(Data\Audio\Cellphone_Ring.wss), 0.75, 3.16228, 1, 75}};
+        GVAR(callConnetTime)[] = {1,8};
```

Comment:
Correct typo wherever necessary:
```suggestion
        GVAR(callConnectTime)[] = {1,8};
```